docs/DeveloperGuide.adoc

```diff
@@ -729,8 +729,60 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |add a contact picture for my contacts |identify them more easily
+
+|`* * *` |user |categorise my contacts into groups |view my contacts by relevant groups
```

Comment:
Maybe give a bit more details about how the contacts will be categorised? By address, date added, tags etc.